src/index.ts

```diff
@@ -25,6 +25,7 @@ export { Rule as NoUnusedCssRule } from './noUnusedCssRule';
 export { Rule as PipeImpureRule } from './pipeImpureRule';
 export { Rule as PipeNamingRule } from './pipeNamingRule';
 export { Rule as PreferOutputReadonlyRule } from './preferOutputReadonlyRule';
+export { Rule as PreferInlineDecorator } from './preferInlineDecoratorRule';
```

Comment:
nit: I guess these exports are alphabetically ordered, so it would come before `preferOutputReadonly`.